src/cc/libbpf.c

```diff
@@ -637,44 +637,48 @@ static int bpf_try_perf_event_open_with_probe(const char *name, uint64_t offs,
 }
 
 static int bpf_attach_tracing_event(int progfd, const char *event_path,
-                                    struct perf_reader *reader, int pid) {
-  int efd, pfd, cpu = 0;
+                                    struct perf_reader *reader, int pid,
+                                    int pfd)
+{
+  int efd, cpu = 0;
   ssize_t bytes;
   char buf[256];
```

Comment:
Not related to your patch, but for long file path, the buffer size may not be enough. Could you help change it to PATH_MAX?

Reply:
Teng just fixed it :)